src/main/kotlin/controller/ImageController.kt

```diff
@@ -0,0 +1,57 @@
+package controller
+
+import javafx.embed.swing.SwingFXUtils
+import javafx.scene.image.Image
+import javafx.scene.image.WritableImage
+import models.IPEwGImage
+import models.ImageModel
+import processing.BasicFilter
+import tornadofx.*
+import java.io.File
+import java.io.IOException
+import java.net.URI
+import javax.imageio.ImageIO
+
+class ImageController : Controller() {
+    lateinit var rawImage: Image
+    lateinit var resultImage: WritableImage
+    val activeImage = ImageModel(IPEwGImage())
+    var isRaw = true
+
+    init {
+        load("./test_image.png")
+        activeImage.image.set(rawImage)
+    }
+
+    fun save(path: String, format: String = "png") {
```

Comment:
I feel like load and save are functionalities of the app, and functionalities belong to the model not the controller. So maybe this fits better in a Model class then we can also unittest it since we mostly write tests for models and not controllers(?) Just my opinion, happy for any counter-argument